tests/Builder/ShowBuilderTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Builder;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\FieldDescriptionCollection;
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+use Sonata\AdminBundle\Builder\ShowBuilderInterface;
+use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
+use Sonata\DoctrineORMAdminBundle\Builder\ShowBuilder;
+use Sonata\DoctrineORMAdminBundle\Model\ModelManager;
+use Symfony\Component\Form\Guess\TypeGuess;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
+ */
+class ShowBuilderTest extends TestCase
+{
+    /**
+     * @var TypeGuesserInterface
```

Comment:
Adding this makes scrutinizer think this is a TypeGuesserInterface, but it is a ObjectProphecy (or something like that). IMO I would remove all those comments from test classes, because they do not give a lot of information, and just make noise

Reply:
my bad, fixed.